### ISSUE TITLE
Why multiply by 0.5 in loss_vsd = 0.5 * F.mse_loss(latents, target, reduction="sum") / batch_size ?

### ISSUE THREAD

**Issue by sen-mao** [threestudio-project/threestudio#519]
Why multiply by 0.5 when calculating VSD loss? Is there a special meaning? Can it be replaced with other numbers, such as 0.1?

https://github.com/threestudio-project/threestudio/blob/28d9d80d9d00f308244adfcf3be8b17ca0cb6465/threestudio/models/guidance/stable_diffusion_vsd_guidance.py#L674

**Comment by AllorH** [comment]
It can't be replaced with other numbers. 
Note that the VSD formulation (Eq. 9 in ProlificDreamer) represents **the gradient for each particle** (NOT the loss function)

If we define the loss as L = 1/2 * ||x - y||², then its **gradient** with respect to 𝑥 is simply **∂L/∂x = (x - y) * dx/dΘ**

In the code, this process corresponds to: `loss = 0.5 * F.mse_loss(latents, target)`
When `loss.backward()` is called, PyTorch automatically computes the **gradient** of the `loss = 0.5 * F.mse_loss(latents, target)`, which corresponds to the formulation described in Eq. (9) of ProlificDreamer.
